client/state/login/selectors.js

```diff
@@ -250,16 +243,15 @@ export const getRequestSocialAccountError = ( state ) =>
  * @returns {?boolean}         Boolean describing social account linking status
  */
 export const getSocialAccountIsLinking = ( state ) =>
-	get( state, 'login.socialAccountLink.isLinking', null );
+	state.login.socialAccountLink?.isLinking ?? null;
 
 /**
  * Gets social account linking email
  *
  * @param  {object}   state  Global state tree
  * @returns {?string}         wpcom email that is being linked
  */
-export const getSocialAccountLinkEmail = ( state ) =>
-	get( state, 'login.socialAccountLink.email', null );
+export const getSocialAccountLinkEmail = ( state ) => state.login.socialAccountLink?.email ?? null;
```

Comment:
Optional chaining not needed, fallback is OK.

Reply:
Done.